src/main/test/ApplicationUtilsTests.cpp

```diff
@@ -368,7 +365,7 @@ class SimulationHelper
 
         // State has been rebuilt and node is properly in sync
         REQUIRE(checkState(*app));
-        REQUIRE(app->getLedgerManager().getLastClosedLedgerNum() ==
+        REQUIRE(app->getLedgerManager().getLastClosedLedgerNum() + 1 >=
```

Comment:
there was a race in this test: it's setup with 1 validator and 1 watcher, and tests that a watcher can get in sync. The thing is that the validator can get ahead of the watcher depending on how much we crank (since the validator does not need the watcher to advance LCL). So I relaxed the condition a bit.

Reply:
gotcha.